crates/telio-model/src/features.rs

```diff
@@ -428,6 +429,13 @@ where
     Ok(Some(eps))
 }
 
+/// Customize firewall policy
+#[derive(Default, Copy, Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureFirewall {
+    /// Custom private IP range
+    pub custom_private_ip_range: Option<Ipv4Network>,
```

Comment:
Creating a separate PR for this https://github.com/NordSecurity/libtelio/pull/605
So that this change is included in v5.0 otherwise this would create a breaking change.